File: dbaasp_api_helper_libraries/ruby/request/PeptideCardRequest.rb
class PeptideCardRequest  < AbstractRequest

    attr_accessor :peptide_id

    def initialize()
       super();
       @peptide_id = 1
    end
    
	def query_type
        return "peptide_card";
    end

    def get_parameters
       dictionary = { "peptide_id" => self.peptide_id}
       return dictionary
    end

end

File: dbaasp_api_helper_libraries/python/request/PeptideCardRequest.py
from AbstractRequest import AbstractRequest

class PeptideCardRequest(AbstractRequest):

    peptide_id = 1;

    def query_type(self):
        return "peptide_card"

    def get_parameters(self):
        dict = {}
        dict["peptide_id"] = self.peptide_id;
        return dict
